sherpa/astro/tests/test_astro_data2.py

```diff
@@ -977,3 +978,46 @@ def test_xmmrgs_notice(make_data_path):
     dat.ignore(10, 19.)
     assert len(dat.get_dep(filter=True)) == 20
     assert dat.get_filter(format='%.2f') == '19.01:19.20'
+
+
+def test_pickle_image_filter_none(make_test_image):
+    """Check we can pickle/unpickle without a region filter.
+
+    This test assumes we have region support, but we do not
+    currently have any test builds without it so do not
+    bother skipping (as many tests would need to be fixed
```

Comment:
I would just remove the text in brackets. The sentence above is clear in itself. If we ever add tests without a no-region build, which we probably should (to quote @DougBurke :-0  ), we will forget to update this as this comment in a test and then it becomes confusing.